src/builder.ts

```diff
@@ -906,12 +906,6 @@ export class SchemaBuilder {
       warn: consoleWarn,
     })
     config.definition(definitionBlock)
-    const toExtend = this.typeExtendMap[config.name]
-    if (toExtend) {
-      toExtend.forEach((e) => {
-        e.definition(definitionBlock)
-      })
-    }
```

Comment:
currently, we cannot extend interface types, so it's safe to remove these codes
https://codesandbox.io/s/extendtype-and-interface-ysf1v?file=/src/schema/index.ts